### ISSUE TITLE
494th SOP - Section numbering

### ISSUE THREAD

**Issue by 132ndNeck** [daviddcs/documentation#5]
For maintainence purposes, should we drop the section numbering?
Example
> 3.2.2. Flight Info

Do github page provide any way of supporting the numbering? (Like in word), or is it all manual editing every time something need to change? If it is manual, it think it may be worth deleting it.

**Comment by daviddcs** [comment]
yes, also it was easy to link to # headings, but they could not contain special characters like . , #¤%&, hence why they are removed 4.1 = 41. linking also requires the headings to be unique.

I will start with removing the X.X.X numbering.
